### DISCUSSION TITLE
Warning message " 'Filter_name' in the input file but not to be taken into account in the fits"

### DISCUSSION THREAD

**Discussion by CamembertGeek** [mboquien/cigale#83] [Q&A] [answered]
Hi, i am curently trying to fit some N2CLS data with CIGALE but i have this warning message when i run CIGALE. Do you have an idea of where comme the probleme ?    

![Capture d’écran du 2024-04-16 14-15-21](https://github.com/mboquien/cigale/assets/162305759/c765546f-5fa4-4430-87fa-1531d5d21aac)

This is my input file:
[source3.txt](https://github.com/mboquien/cigale/files/14996638/source3.txt)

**Comment by mboquien** [comment, marked as answer]
Bonjour. There things happening here. First, the input flux file does not contain the columns named galex.FUV, galex.NUV, spitzer.MIPS.24, SCUBA450 et VLA_L. They need to be renamed, this is easily done with topcat or astropy. The other thing is that the grid contains a single model. The sampling should be given in the pcigale.ini file for each parameter that is free. CIGALE then builds a Cartesian grid. The number of models can vary greatly depending on the exact modeling, but something of the order of the million is quite typical.
